spec/spec_helper_acceptance_local.rb

```diff
@@ -1,53 +0,0 @@
-def ldapsearch(cmd, exit_codes = [0, 1], &block)
-  puts "shell() not working in litmus for now"
-  #shell("ldapsearch #{cmd}", acceptable_exit_codes: exit_codes, &block)
-end
-
-RSpec.configure do |c|
-  c.before :suite do
-    run_shell('puppet cert generate $(facter fqdn) || true')
-    pp = <<-EOS
-     $ssldir = '/etc/puppetlabs/puppet/ssl'
-     file { '/etc/ldap':
```

Comment:
This setup is no longer needed? Pretty sure you're already aware of this, but https://github.com/voxpupuli/voxpupuli-acceptance/#per-host-configuration.

Reply:
It was moved around (where it will be overwritten by modulesync), I though about the voxpupuli-acceptance thing but was focusing on something else so forgot about it.  There is also a bunch of functions that should be moved out of modulesync's way.  Thanks!